docs/01-development-guide.md

```diff
@@ -1,25 +1,81 @@
 ---
 title: Development Guide
 expires_at: never
-tags: [routing-release,gorouter]
+tags: [ routing-release,gorouter ]
 ---
 
-<!-- vim-markdown-toc GFM -->
+# Development Guide
 
-* [Development Guide](#development-guide)
-  * [Golang TCP Networking Basics](#golang-tcp-networking-basics)
-  * [General Gorouter Architecture](#general-gorouter-architecture)
-  * [Ifrit processes](#ifrit-processes)
-  * [What does Gorouter do?](#what-does-gorouter-do)
-  * [What are all these extra components in the Gorouter request flow?](#what-are-all-these-extra-components-in-the-gorouter-request-flow)
-    * [Negroni Handlers](#negroni-handlers)
-    * [ProxyRoundTripper](#proxyroundtripper)
+## Reporting issues and requesting features
 
-<!-- vim-markdown-toc -->
+Please report all issues and feature requests in [cloudfoundry/routing-release](https://github.com/cloudfoundry/routing-release).
 
-# Development Guide
+## Contributing
```

Comment:
@a18e This information along with how to run tests or build binaries is all in the main README.me under Contributing. I would stay away from repeating this information in the development-guide file.

Reply:
Yep, you're right. Removed the Contributing section. 
But the content of the `Setup` section is actually quite different from `CONTRIBUTING.md` since it's gorouter specific.